### ISSUE TITLE
Research: Look into using docs-json output target to improve Storybook integration

### ISSUE THREAD

**Issue by jcfranco** [Esri/calcite-components#1079]
@jcfranco commented on [Fri Feb 14 2020](https://github.com/Esri/calcite-app-components/issues/826)

### Background 

Stencil 5.3 added support for web components, namely [`custom-elements.json`](https://github.com/storybookjs/storybook/tree/next/addons/docs/web-components#custom-elementsjson). This could be used in conjunction with Stencil's [JSON doc output](https://stenciljs.com/docs/docs-json) to help simplify our setup.

### Desired Outcome 

Streamline the Stencil doc output + Storybook integration.

---

@jcfranco commented on [Sat Feb 15 2020](https://github.com/Esri/calcite-app-components/issues/826#issuecomment-586640057)

This should probably wait until https://github.com/ionic-team/stencil/issues/2196. The current JSON doc output from Stencil does not give a lot of info for Storybook to work its magic.

**Comment by julio8a** [comment]
Does this relate to the work @caripizza did?

**Comment by jcfranco** [comment]
The Stencil team is looking into improving Storybook integration. I'll leave this open and revisit once there's an update. Related issues:

https://github.com/ionic-team/stencil/issues/3104
https://github.com/ionic-team/stencil/issues/2328
https://github.com/storybookjs/storybook/issues/4600

**Comment by jcfranco** [comment]
This is no longer relevant after #10310. Closing!
